Modules/Data.lua

```diff
@@ -483,5 +483,18 @@ data.uniques = { }
 for _, type in pairs(itemTypes) do
 	data.uniques[type] = LoadModule("Data/Uniques/"..type)
 end
+data.uniqueMods = { }
+data.uniqueMods["Watcher's Eye"] = { }
+local unsortedMods = LoadModule("Data/Uniques/Special/WatchersEye")
+local sortedMods = { }
```

Comment:
You can do this in one step by passing a custom comparison function to `table.sort`

Reply:
See https://www.programming-idioms.org/idiom/28/sort-by-a-property/1341/lua for an example.

Reply:
Actually, that doesn't work here because the comparison function doesn't get the key, only the value, and since `modId` only appears as the key, I have to do this or use an alternative `pairs` iterator that iterates by key.

Reply:
Looks like the docs suggesting to make a new iterator function do this behind the scenes anyway: https://www.lua.org/pil/19.3.html

Reply:
I see. We could consider adding such a function if this special case comes up more often in the future.